test/basic-sell.ts

```diff
@@ -36,6 +36,7 @@ describe('BasicSellCommand', () => {
         const data = system.automationBot.interface.encodeFunctionData('addTrigger', [
             testCdpId,
             TriggerType.BASIC_SELL,
+            false,
```

Comment:
this should be an argument in the function
`createTrigger = async (triggerData: BytesLike, continuous: boolean)`
otherwise it's false for all the tests

Reply:
also the last test can be changed to `'doesn`t remove the trigger if `continuous` is set to true'` when the bb and bs contracts are changed